stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -154,7 +154,7 @@ end
 export.set_to_zero
     padw
     repeat.512
-        dup.4 add.1 swap.5
+        dup.4 add.4 swap.5
         mem_storew
     end
```

Comment:
Similar comment as above re updating the cycle count.

Reply:
Updated to ~3100